add_events.py

```diff
@@ -112,8 +112,12 @@ def main():
                 replaceHour += int(startHour)
             event = {}
             # Currently there are no labs in `subjects.json`
-            if (data[day][time][0] in subjects.keys()):
-                event['summary'] = "Class of " + subjects[data[day][time][0]].title()
+            # if (data[day][time][0] in subjects.keys()):
+            if (data[day][time][2] < 3):
```

Comment:
Can there be a lab of less than 3 hours ?
It seems logical as per our experience, but is it correct!